src/libs/agent-runtime/cloudflare/index.ts

```diff
@@ -0,0 +1,127 @@
+import { ChatModelCard } from '@/types/llm';
+
+import { LobeRuntimeAI } from '../BaseAI';
+import { AgentRuntimeErrorType } from '../error';
+import { ChatCompetitionOptions, ChatStreamPayload, ModelProvider } from '../types';
+import {
+  CloudflareStreamTransformer,
+  DEFAULT_BASE_URL_PREFIX,
+  convertModelManifest,
+  desensitizeCloudflareUrl,
+  fillUrl,
+} from '../utils/cloudflareHelpers';
```

Comment:
no need to add cloudflareHelpers to the utils. the openai helper and anthropic helper are placed there because there are reused in different provider implement. I think these utils are only used by cloudflare.